python/feldera/pipeline.py

```diff
@@ -394,20 +308,18 @@ def shutdown(self):
                 # block until the callback runner has been stopped
                 queue.join()
 
-        self.client.shutdown_pipeline(self.pipeline_name)
+        self.client.shutdown_pipeline(self.name)
 
     def resume(self):
         """
         Resumes the pipeline.
         """
 
-        self.client.start_pipeline(self.pipeline_name)
+        self.client.start_pipeline(self.name)
 
     def delete(self):
```

Comment:
`delete()` should have an optional argument `shutdown=False`

Reply:
When set True, that would shutdown and delete?

Reply:
If the pipeline is not shutdown, deletion is not possible... Hmm, I mean it would just be a shorthand for:
```
pipeline.shutdown()
pipeline.delete()
```
It can also just be kept as-is, demo scripts would just be 1 line longer which is also fine.